src/hotspot/share/utilities/numberSeq.hpp

```diff
@@ -25,7 +25,7 @@
 #ifndef SHARE_UTILITIES_NUMBERSEQ_HPP
 #define SHARE_UTILITIES_NUMBERSEQ_HPP
 
-#include "memory/allocation.hpp"
+#include "utilities/ostream.hpp"
```

Comment:
`CHeapObj` is from allocation.hpp. So the original include is correct.

Instead of including the header just for `outputStream`, better forward-declare it in the .hpp file here and include it in the cpp file.